Tasks/UseDotNetV2/README.md

```diff
@@ -36,6 +36,8 @@ Examples:
   - To install latest patch version of 2.1 sdk, use 2.1.x
   - To install latest minor version of 2. sdk, use 2.x<br/>For getting more details about exact version, refer [this link](https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json).
 
+* **Compatible Visual Studio Version**: Specify version of compatible visual studio for which .NET core sdk to install. Specifiy complete vs-version like 16.6.4 containing major version, minor version and patch number. Find the value of `vs-version` for installing SDK/Runtime, from the releases.json. The link to releases.json of that major.minor version can be found in [**releases-index file.**](https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json)
```

Comment:
`Specify a compatible Visual Studio version. If a version is specified, then only those versions of .Net Core SDK/runtime will be considered for installation which are compatible with this Visual Studio version. Specify complete Visual Studio version containing major version, minor version and patch number. e.g. 16.6.4.
Compatibility of Visual Studio is checked by looking at `vs-version` property specified in the release manifest from the releases.json files. The link to releases.json of that major.minor version can be found in [**releases-index file.**](https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json)`